docs/fiddles/features/drag-and-drop/main.js

```diff
@@ -0,0 +1,35 @@
+const { app, BrowserWindow, ipcMain } = require('electron')
+
+function createWindow () {
+  const win = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      nodeIntegration: true
+    }
+  })
+
+  win.loadFile('index.html')
+  win.webContents.openDevTools()
+}
+
+app.whenReady().then(createWindow)
+
+ipcMain.on('ondragstart', (event, filePath) => {
+  event.sender.startDrag({
+    file: filePath,
+    icon: '/path/to/icon.png'
```

Comment:
I think this is giving me the following error since this is still an example path so the demo isn't runnable.
![image](https://user-images.githubusercontent.com/16010076/99608797-78624600-29c3-11eb-95d6-763348e26d06.png)